src/d3a/resources/Solar_Curve_W_sunny_normalized.csv

```diff
@@ -0,0 +1,97 @@
+INTERVAL;POWER(W)
+00:00;0.0
+00:15;0.0
+00:30;0.0
+00:45;0.0
+01:00;0.0
+01:15;0.0
+01:30;0.0
+01:45;0.0
+02:00;0.0
+02:15;0.0
+02:30;0.0
+02:45;0.0
+03:00;0.0
+03:15;0.0
+03:30;0.0
+03:45;0.0
+04:00;0.0
+04:15;0.0
+04:30;0.0
+04:45;0.0
+05:00;0.0
+05:15;0.05
+05:30;0.05
+05:45;0.05
+06:00;0.05
+06:15;0.15
+06:30;0.15
+06:45;0.15
+07:00;0.15
+07:15;0.3
+07:30;0.3
+07:45;0.3
+08:00;0.3
+08:15;0.5
+08:30;0.5
+08:45;0.45
+09:00;0.5
+09:15;0.65
+09:30;0.55
+09:45;0.65
+10:00;0.6
+10:15;0.75
+10:30;0.65
+10:45;0.7
+11:00;0.65
+11:15;0.8
+11:30;0.85
+11:45;0.8
+12:00;0.8
+12:15;0.7
+12:30;0.65
+12:45;0.75
+13:00;0.75
+13:15;0.6
+13:30;0.7
+13:45;0.65
+14:00;0.7

```

Comment:
@chrisd12 Please have a look at the weight profiles here which I derived from dividing the exiting predefined PV profiles by 250 (as you suggested in the ticket D3ASIM-3493). However, the maximum value was 200W in the sunny profile. So, If you scale a sunny profile with `capacity_kW=250`, you will only get a peak production of 200W for the sunny case (and less for the cloudy and partially cloudy cases). Do you see this as a problem?

Reply:
@hannesdiedrich I don't see it as a problem at all. Usually when a PV manufacturer state the capacity (e.g. 250W) it means the maximum power the panel can produce under perfect conditions (irradiance, angle, temperature) and these are very rarely met. So it is fine to have an output a little lower than the capacity.
I have tested a couple of simulation and it seems to be working. Just one thing, can you please rename the normalised csv files to kW (e.g. `Solar_Curve_W_cloudy_normalized` --> `Solar_Curve_kW_cloudy_normalized`)?

Reply:
@chrisd12  Ok, thanks a lot! Makes a lot of sense!
RWT the renaming: I dont agree and would even change it to: `Solar_Curve_cloudy_normalized` as the weights in there dont have any unit (I will also remove the unit form the table header). Right?

Reply:
@hannesdiedrich The profile is multiplied by the capacity_kW. If capacity_kW=1 then the profile remained unchanged. If capacity_kW=2 then the profile values are multiplied by 2. Basically this file is a normalised profile but it is also a profile of a PV with a capacity of 1kW.

Reply:
@chrisd12 I am not convinced. Indeed, `it is also a profile of a PV with a capacity of 1kW`, but only if you multiply by `1 kW`. Sorry, but my physics professor would agree with me, these weights are unitless and you only make a power profile out of it, when you multiply `capacity_kW` . 
If you want to say that these are 1kW profiles, then we need to rename the scaling value to only `capacity_scale` or so.

Reply:
@hannesdiedrich ok then let's proceed **without** the unit. Please name it `Solar_Curve_cloudy_normalized`. The column name `Profile weigh` looks good to me